modules/random_cat_fact.py

```diff
@@ -0,0 +1,11 @@
+import requests
+import json
+
+def random_cat_facts():
+    try:
+        url = 'https://cat-fact.herokuapp.com/facts/random'
+	response = request.get(url)
+	response_json = json.loads(response.text)
+	return = "Cat Fact: " + response_json.get('text')
```

Comment:
SyntaxError: invalid syntax
```suggestion
	return "Cat Fact: " + response_json.get('text')
```